client/src/actions/types.js

```diff
@@ -27,3 +27,5 @@ export const CLEAR_ALL_PET_PROFILES = 'CLEAR_ALL_PET_PROFILES';
 
 export const GET_ADMINS_PET_PROFILES = 'GET_ADMINS_PET_PROFILES';
 export const CLEAR_ADMINS_PET_PROFILES = 'CLEAR_ADMINS_PET_PROFILES';
+
+export const GET_LATEST_PET_UPDATE_PROFILES = 'GET_LATEST_PET_UPDATE_PROFILES';
```

Comment:
Since we just return the pet status info but not the whole pet profile, so I am thinking maybe we can rename this. How about a name like this: `GET_LATEST_PET_STATUS` ?